src/components/ListItem.test.js

```diff
@@ -1,39 +1,47 @@
 import React from 'react';
 import ListItem from './ListItem';
-import { mount } from 'enzyme';
-import configureMockStore from 'redux-mock-store';
-import thunk from 'redux-thunk';
-import { Provider } from 'react-redux';
+import { shallow } from 'enzyme';
 
-const mockStore = configureMockStore([thunk]);
+describe('ListItem component (test independent from React DnD)', () => {
+  let component;
+
+  // Reference for this solution: https://github.com/react-dnd/react-dnd/issues/925
+  const getComponent = (item = {}) => {
+    // Obtain the reference to the component before React DnD wrapping
+    let OriginalComponent = ListItem.DecoratedComponent;
+    // Stub the React DnD connector functions with an identity function
+    let identity = el => el;
+
+    if (!component) {
+      component = shallow(
+        <OriginalComponent
+          item={item}
+          key={item.id}
+          connectDragSource={identity}
+          connectDropTarget={identity}
+        />
+      );
+      return component;
+    }
+  };
 
-describe('ListItem component', () => {
-  let store;
   beforeEach(() => {
-    store = mockStore({});
+    component = undefined;
   });
+
   test('renders ListItem without crashing without a metadata item', () => {
-    const wrapper = mount(
-      <Provider store={store}>
-        <ListItem />
-      </Provider>
-    );
-    expect(wrapper.find('LisItemControls')).toBeDefined();
-    expect(wrapper.find('ListItemEditForm')).toBeDefined();
+    const wrapper = getComponent({});
+    expect(wrapper.instance().props.item).toEqual({});
```

Comment:
Ah, so that `wrapper.instance()` does the trick eh?

Reply:
Yeah. Earlier we had to find the relevant component in there to match the props.